src/styles/app/25-node-note-content.css

```
@@ -581,32 +581,68 @@
 }
 
 .tiptapContentInline mark[data-glyph-highlight] {
-	background: transparent;
-	padding: 0.02em 0.12em;
+	--glyph-highlight-marker-color: var(
+		--glyph-inline-highlight-yellow,
+		rgba(240, 180, 41, 0.26)
+	);
+	--glyph-highlight-marker-grain: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='256' height='64'%3E%3Cdefs%3E%3Cfilter id='g' x='0' y='0' width='100%25' height='100%25' color-interpolation-filters='sRGB'%3E%3CfeTurbulence type='fractalNoise' baseFrequency='0.04 0.7' numOctaves='1' stitchTiles='stitch' seed='3' result='s'/%3E%3CfeColorMatrix in='s' values='0 0 0 0 0  0 0 0 0 0  0 0 0 0 0  0 0 0 .18 .72' result='sa'/%3E%3CfeTurbulence type='fractalNoise' baseFrequency='0.012' numOctaves='2' stitchTiles='stitch' seed='7' result='p'/%3E%3CfeColorMatrix in='p' values='0 0 0 0 0  0 0 0 0 0  0 0 0 0 0  0 0 0 .16 .76' result='pa'/%3E%3CfeComposite in='sa' in2='pa' operator='arithmetic' k1='1' k2='0' k3='0' k4='0'/%3E%3C/filter%3E%3C/defs%3E%3Crect width='256' height='64' fill='white' filter='url(%23g)'/%3E%3C/svg%3E");
+	background-color: transparent;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)